diesel_derives/src/lib.rs

```diff
@@ -394,10 +396,10 @@ pub fn derive_query_id(input: TokenStream) -> TokenStream {
 ///
 /// This trait can only be derived for structs, not enums.
 ///
-/// **When this trait is derived, it will assume that the order of fields on your
-/// struct match the order of the fields in the query. This means that field
-/// order is significant if you are using `#[derive(Queryable)]`. Field name has
-/// no effect.**
+/// **Note**: When this trait is derived, it will assume that __all fields on
+/// your struct__ matches __all fields in the query__, including the order and
+/// count. This means that field order is significant if you are using
+/// `#[derive(Queryable)]`. Field name has no effect.
```

Comment:
I would like to keep the part with the field names bold here. That's the point people miss quite often.

Reply:
I personally find bolding everything hard to read, so that is why I bold only the first **Note** part. Because I could also miss it when the full text is bold but it doesn't seemed like something important without the special **Note** text. Do you want me to change it back?

Reply:
That's correct, therefore I would only change the last sentence to bold.
```suggestion
/// **Note**: When this trait is derived, it will assume that __all fields on
/// your struct__ matches __all fields in the query__, including the order and
/// count. This means that field order is significant if you are using
/// `#[derive(Queryable)]`. __Field name has no effect__.
```